src/axom/mir/detail/elvira_impl.hpp

```diff
@@ -1005,7 +1005,7 @@ AXOM_HOST_DEVICE void missvol1(FloatType c00,  // Center column sum.
                                FloatType c01,
                                FloatType v01,
                                FloatType &vma,
-                               int far)
+                               int is_far)
```

Comment:
`far` appears to be a macro on Windows :(

Reply:
Ah, NEAR/FAR. Too bad Windows CI did not catch this _(must not have RAJA/UMPIRE/Conduit)_. Thanks for fixing it.

Reply:
Our current Windows CI for PRs only tests the parts of Axom that don't realy on any TPLs. We also have a manual CI job to test Axom w/ TPLs  (https://github.com/LLNL/axom/actions/workflows/test_windows_tpls.yml).

With the update to github actions for our CI, it looks like it'll be a lot easier to cache out Windows TPLs, which would allow us to significantly improve our Windows testing.